### PR TITLE
Some typos + fixing me OCD

### PR DESCRIPTION
**Pull request by MiTonder**
I translated some minor typos and made chances to the navigation bar so that every word starts with a big letter because my OCD was triggering everytime i was om fnbr.co

### CODE REVIEW

**Comment by Krymonota** [comment]
In fact, using lowercase letters in the navigation menu is more of a design question than a spelling question. Although capitalizing the first letter would be orthographically correct, other languages also write it in lowercase because the website layout is designed that way. So I won't merge this pull request in its current state.

**Comment by MiTonder** [comment]
NOW?

File: da.json [hunk 1]
```diff
@@ -69,9 +69,9 @@
                 "translate" : "oversæt",
                 "about" : "om fnbr.co",
                 "privacy" : "fortrolighedspolitik",
-                "donate" : "Donere",
-                "discord" : "Discord",
-                "twitter" : "Twitter"
+                "donate" : "donere",
```
Comment (Krymonota) [inline da.json]: According to the English string, the first letter of this string must be capitalized.

File: da.json [hunk 2]
```diff
@@ -69,9 +69,9 @@
...
+                "discord" : "discord",
```
Comment (Krymonota) [inline da.json]: According to the English string, the first letter of this string must be capitalized. (Brands have to be capitalized anyway.)

File: da.json [hunk 3]
```diff
@@ -69,9 +69,9 @@
...
+                "twitter" : "twitter"
```
Comment (Krymonota) [inline da.json]: According to the English string, the first letter of this string must be capitalized. (Brands have to be capitalized anyway.)

**Comment by MiTonder** [comment]
I think i fixed it

**Pull request merged**